common-items.lic

```diff
@@ -356,6 +356,7 @@ module DRCI
 
     trashcans = DRRoom.room_objs
                       .reject { |obj| obj =~ /azure \w+ tree/ }
+                      .map { |obj| obj.gsub('small bin concealed with some nearby brush', 'small bin') }
```

Comment:
Does `DRC.get_noun` not work on "small bin concealed with some nearby brush"?

Reply:
It returns `concealed` so I implemented a workaround suggestion form @asechrest

Reply:
Confirming it doesn't pare it down to the correct noun. (Interestingly, @KatoakDR 's noun regex refactor works perfectly on it, but I don't know that it's ready for release).

We threw this around in Discord quite a bit today. I suggested this solution as a way to just target this specific bin and use `gsub` to get it into a form where the noun regex would work on it, and then the rest of it flows fine.

I welcome better approaches, though.

Also appreciate @MazMedias submitting the PR!

Reply:
I think I have a fix for the current noun parsing regex at https://github.com/rpherbig/dr-scripts/pull/5827

Interim fix until I can spend more time to get https://github.com/rpherbig/dr-scripts/pull/5435 ready for primetime